### ISSUE TITLE
cannot insert placeable items into spirt table 

### ISSUE THREAD

**Issue by Ratbagdoo** [SammySemicolon/Malum-Mod#260]
I do not know how to explain my issue but when i try to make the fabric I cannot insert the string or wool because it's being placed instead.

**Comment by Ratbagdoo** [comment]
It appears to be an incompatibility with either crafttweaker and or Jei Tweaker

**Comment by Ratbagdoo** [comment]
upon further testing it appears to be an server issue.

**Comment by Ratbagdoo** [comment]
funny enough just replacing the block temporally fixes the issue

**Comment by f-raZ0R** [comment]
Having this issue on a server as well, seems to be a client-server desync, every time i craft an item i then have to break and replace the altar and its incredibly annoying. Please look into this more.

**Comment by f-raZ0R** [comment]
okay, so what happens seems to be a client-server desync, where the client thinks there isn't an item in the altar, but the server thinks there is.
this can also happen vice versa, with pedestals, with the client thinking there's an item there, but the server thinking there isn't.
When THIS happens, the craft animation starts, but nothing happens, and it spams the server log with contextless STDOUTs { >:(  } saying the item ids of whats "on the pedestals".

**Comment by f-raZ0R** [comment]
Encountering this on latest fabric version, 1.20.1, can't exactly test whether this is a mod conflict or if it happens with just malum, as i'm not the one who owns the server and who can choose what mods are installed

**Comment by f-raZ0R** [comment]
Seems like this is due to https://github.com/SammySemicolon/Malum-Mod/blob/1.6-1.20.1/src/main/java/com/sammy/malum/common/block/curiosities/runic_workbench/RunicWorkbenchBlockEntity.java only updating the item's stack size for the server?
so the client still thinks there's an item on there, and won't let you place another one?
idk, a bit hard to follow due to all the other functions this calls, but i think it has something to do with this only happening on the server, and  not the client, causing issues when the server and client aren't the same

**Comment by f-raZ0R** [comment]
Client goes "well, as far as im concerned there's still an item in the primary input slot. so that means you're gonna try to put that item into the secondary input instead", even though the server is saying "no there's nothing there"

**Comment by f-raZ0R** [comment]
never mind ignore me this is the wrong block
